js/company.js

```diff
@@ -0,0 +1,48 @@
+/* variables */
+/* only the symbol by splitting the url */
+const symbol = window.location.search.split("=")[1];
```

Comment:
it's better to use URLSearchParams. see examples here:
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#examples